src/prisma/schema.prisma

```diff
@@ -38,3 +40,16 @@ model Profile {
   website                  Json?
   tdaGradYearBannerVisible Json
 }
+
+model Post {
+  id                 Int   @id @default(autoincrement())
+  userId             Int?   @unique
+  user               User?  @relation("UserToPost",fields: [userId], references: [id])
```

Comment:
After references, let's add `, onDelete: Cascade` so that if any post is deleted, comments or anything else depending on it will also be deleted.